rust_entrypoint/build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("cargo:rustc-link-search=../program/target");
```

Comment:
Does not work when I do. This tells the Rust Build system where to look for the archive file that includes the C code. It might be possible to bypass that if we put the archive file in a certain location, but this feels more readable/standard to me.